dawn/src/dawn/IIR/ASTConverter.cpp

```diff
@@ -67,9 +67,9 @@ void ASTConverter::visit(const std::shared_ptr<sir::VerticalRegionDeclStmt>& stm
           stmtMap_.at(stmt->getVerticalRegion()->Ast->getRoot()))),
       stmt->getVerticalRegion()->VerticalInterval, stmt->getVerticalRegion()->LoopOrder,
       stmt->getVerticalRegion()->Loc);
-
-  stmtMap_.emplace(stmt,
-                   iir::makeVerticalRegionDeclStmt(verticalRegion, stmt->getSourceLocation()));
+  verticalRegion->iterationSpace_ = stmt->getVerticalRegion()->iterationSpace_;
```

Comment:
Unrelated, but could you explain to me what does this visitor method does?

Reply:
Converts an AST with SIR data to one with IIR data.